src/test/java/next/reflection/ReflectionTest.java

```diff
@@ -5,14 +5,65 @@
 import org.slf4j.LoggerFactory;
 
 import java.lang.reflect.Constructor;
+import java.text.MessageFormat;
+import java.util.Arrays;
+import java.util.stream.Collectors;
 
 public class ReflectionTest {
     private static final Logger logger = LoggerFactory.getLogger(ReflectionTest.class);
 
+    enum Modifier {
+        PRIVATE(2), PUBLIC(1), PROTECTED(3), UNKNOWN(-1);
+
+        private final int value;
+
+        Modifier(int value) {
+            this.value = value;
+        }
+
+        public static Modifier valueOf(int value){
+            return Arrays.stream(Modifier.values())
+                    .filter(modifier -> modifier.value == value )
+                    .findFirst()
+                    .orElse(UNKNOWN);
+        }
+    }
+
     @Test
     public void showClass() {
         Class<Question> clazz = Question.class;
+
         logger.debug(clazz.getName());
+
+        // 생성자
+        Arrays.stream(clazz.getDeclaredConstructors())
+                .forEach(constructor -> {
+                    logger.debug(MessageFormat.format("constructors: {0} {1} ({2})",
+                            Modifier.valueOf(constructor.getModifiers()).name(),
+                            clazz.getName(),
+                            Arrays.stream(constructor.getParameterTypes()).map(it -> it.getName()).collect(Collectors.joining(" ,"))));
```

Comment:
```suggestion
                            Arrays.stream(constructor.getParameterTypes()).map(it -> it.getName()).collect(Collectors.joining(", "))));
```
[의견] ~조금 짜치지만~ 이게 맞을것 같아요 😅

Reply:
네, 그게 맞네요 😅